markdownhighlighter.cpp

```diff
@@ -73,7 +73,7 @@ void MarkdownHighlighter::timerTick() {
     // emit a signal every second if there was some highlighting done
     if (_highlightingFinished) {
         _highlightingFinished = false;
-        emit highlightingFinished();
+        Q_EMIT highlightingFinished();
```

Comment:
@Waqar144, are you ok with the Q_EMITs?

Reply:
Yes, it's better. However I would personally suggest you to convert them all in one go, maybe in a separate commit/PR. emit, signals, slots etc